README.md

```
@@ -1,56 +1,114 @@
+#### <img width="30" height="30" src="https://github.com/user-attachments/assets/39297c45-f22a-48a1-a892-ff01c2dddb63" /> 원하는 브랜드만 쏙쏙! 완벽한 네컷 지도
+지금 바로 찍고 싶을 때, 주변의 사진관을 확인하세요. 브랜드 필터를 사용해 내가 선호하는 브랜드만 골라 찾을 수 있습니다.
+#### <img width="30" height="30" src="https://github.com/user-attachments/assets/ee60a2a4-b933-447f-9a43-df38676b5fe4" /> 추억을 테마별로 관리
+친구, 연인, 특별한 기념일 등 목적에 맞춰 폴더를 만들고 소중한 순간들을 테마별로 기록하세요.
 
+<!-- 기능 소개 PPT 추가 예정 -->
+
+<br>
+
+## <img width="40" height="40" src="https://github.com/user-attachments/assets/06940c34-fa04-461f-acd6-f6e121ffdd61" /> Contributors
+
+| <img width="200px" height="3400" src="https://avatars.githubusercontent.com/u/99116619?v=4" /> | <img width="200px" height="3400" src="https://avatars.githubusercontent.com/u/63261054?v=4" /> |
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)